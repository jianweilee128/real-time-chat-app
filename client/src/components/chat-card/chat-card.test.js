import { shallow } from "enzyme";
import React from "react";
import ChatCard from "./chat-card.component";

describe("ChatCard", () => {
  it("should render correctly", () => {
    const sender = "test";
    const timestamp = "1 Jan 2020";
    const message = "Hi";
    const component = shallow(
      <ChatCard sender={sender} timestamp={timestamp} message={message} />
    );
    expect(component).toMatchSnapshot();
  });
});
