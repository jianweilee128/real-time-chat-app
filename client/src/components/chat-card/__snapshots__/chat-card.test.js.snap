// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatCard should render correctly 1`] = `ShallowWrapper {}`;
